vizro-core/tests/unit/vizro/models/test_page.py

```diff
@@ -95,7 +95,7 @@ def test_set_layout_invalid(self):
     def test_check_for_valid_component_types(self):
         with pytest.raises(
             ValidationError,
-            match=re.escape("(allowed values: 'button', 'card', 'container', 'graph', 'table')"),
+            match=re.escape("(allowed values: 'button', 'card', 'container', 'graph', 'table', 'tabs')"),
```

Comment:
i hate it 😂  but let's keep it ❤️

Reply:
Judging by what @huong-li-nguyen says I guess this has been discussed at length before 😅 I guess options are, in order of strongest to weakest test:
1. test the whole error message
2. test the part of the error message that needs to be updated when new components are added
3. test the part of the error message that doesn't need to be updated when new components are added
4. remove the test entirely

Currently we're doing option 2. There are reasonable arguments in favour of each option and I don't have a strong opinion on which to do. Personally I probably would have gone for option 3, but given we're currently following option 2 in various places and it's very little effort to maintain this approach I would keep doing as it is.

If you're interested a while ago I came across this issue which discusses similar things and has various opinions: https://github.com/pytest-dev/pytest/issues/10085.